main.py

```diff
@@ -29,6 +29,7 @@ def main():
         menu.create()
 
     #wait for application to exit
+    conn.close()
```

Comment:
I don't think this will work when the application actually uses the database.  This will close it before it waits for the application to exit.  Closing the connection will probably need to be done in the closing event of the main window.